packages/maskbook/src/extension/options-page/DashboardComponents/ContactLine.tsx

```diff
@@ -70,9 +70,9 @@ export function ContactLine(props: ContactLineProps) {
         onDeleted,
     })
 
-    const [menu, openMenu] = useMenu(
+    const [menu, openMenu] = useMenu([
```

Comment:
since add the second param `anchorSibling` to `useMenu()`, it has to pass an array as first param now.